qucs-doc/tutorial/textmode/content.tex

```diff
@@ -151,7 +151,7 @@
 
 \begin{figure}[h]
 \begin{center}
-  \includegraphics[angle=0,width=0.8\linewidth]{rc_ac_python.eps}
```

Comment:
removing the `.eps` extensions was done also in 110a4f4906a62da5b1eabbbe339402b19b04ae0d , which  you already merged; for some reason the change got lost (??).
It seems there are some recent versions `pdflatex` that can use `eps` file but also mine here cannot.